### DISCUSSION TITLE
How to install updates

### DISCUSSION THREAD

**Discussion by Lycidias93** [bucanero/pkgi-ps3#109] [Q&A]
How can I install updates after download? I saw the toast notification from the background download but no idea how to install it

**Comment by Lycidias93** [comment]
I also have no clue in which directory the full games are downloaded or how to install them...

**Comment by bucanero** [comment]
> How can I install updates after download? I saw the toast notification from the background download but no idea how to install it

you should find downloaded packages under the "Package manager" XMB menu. There's an option for network packages and all the bubbles should be there.
